### PR TITLE
Update native modules & Version

### PR DESCRIPTION
**Pull request by kpodemski**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 1.7.7.x
| Description?      | new build
| Type?             | improvement
| Category?         | PM
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | n/a
| How to test?      | n/a
| Possible impacts? | Please indicate what parts of the software we need to check to make sure everything is alright.
| Sponsor company | impSolutions

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/22554)

### CODE REVIEW

**Review by jolelievre — DISMISSED** [review]

File: composer.lock [hunk 1]
```diff
@@ -10012,5 +10016,5 @@
     "platform-overrides": {
         "php": "7.1.3"
     },
-    "plugin-api-version": "2.0.0"
```
Comment (matks) [inline composer.lock]: I dont know if this is OK 🤔  @PierreRambaud what did we say about `plugin-api-version` ?

**Review by matks — DISMISSED** [review]
PR seems OK but I'm not sure about `plugin-api-version`

**Comment by jolelievre** [comment]
> PR seems OK but I'm not sure about `plugin-api-version`

I'm not sure but I believe it depends on which version of composer you use while building the lock But we are compatible with v1 and v2 anyway so I don't think it has much impact

**Comment by matks** [comment]
> > PR seems OK but I'm not sure about `plugin-api-version`
> 
> I'm not sure but I believe it depends on which version of composer you use while building the lock But we are compatible with v1 and v2 anyway so I don't think it has much impact

What I am afraid is, if we put `plugin-api-version: 2`, that people using Composer v1 cannot install PS anymore 🤔  (or the other way around)

File: composer.lock (hunk 1 above)
Comment (kpodemski) [reply composer.lock]: @matks it's because I'm running on v1 locally I think I'll revert it manually?

**Review by jolelievre — APPROVED** [review]

**Review by atomiix — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Comment by matks** [comment]
Thank you @kpodemski

**Pull request merged**